sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelAnnexTest.java

```diff
@@ -121,6 +121,9 @@ void testRequirementB6() throws NoTestData, IOException {
                 acceptableSequenceSeen.get(), "No AbstractOperationDescriptors seen during test run, test failed.");
     }
 
+    // NOTE: No PreCondition can ensure that an ApprovedJurisdictions list is present or absent as this
+    //       list is supposed to be static for a Mds. When all Mds in a Device's Mdib have an ApprovedJurisdictions
+    //       list, then this test case should not be applied to it.
```

Comment:
This is definitely not the right place to "inform" anyone when not to apply the test, no user will ever read it.

Reply:
Made sure this is also stated in RQM.